### PR TITLE
Setup Clinic DataBase

### PR DESCRIPTION
**Pull request by Tchilo**
- Create clinic database ✔️ 
- Add patients table ✔️
- Add medical_histories table ✔️
- Add invoices table ✔️
- Add treatments table ✔️
- Add invoice_items table ✔️
- CREATE INDEX patient_id ON medical_histories(patient_id); ✔️
- CREATE INDEX medical_history_id ON invoices(medical_history_id); ✔️
- CREATE INDEX invoice_id ON invoice_items(invoice_id); ✔️
- CREATE INDEX treatment_id ON invoice_items(treatment_id); ✔️
- Update README.md ✔️

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+create table invoice_items(
+id int primary key generated always as IDENTITY,
+unit_price decimal not null,
+quantity int not null,
+total_price decimal not null,
+invoice_id int,
+treatment_id int,
+constraint invoice_id foreign key(invoice_id) references invoices(id),
+constraint treatment_id foreign key(treatment_id) references treatments(id)
+);
```
Comment (descholar-ceo) [inline schema_based_on_diagram.sql:39]: - [x] All "join table" for the many-to-many relastionships is created, even if they are not showed on the diagram. There fore you should create a table between medical_histories and treatments  (name can be different)

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @Tchilo and @azy64 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (Tchilo) [reply schema_based_on_diagram.sql:39]: thanks for the code review we have made the changes 😃 .

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉🍾✔**
Hello @Tchilo 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**